controller-server/src/main/java/com/yahoo/vespa/hosted/controller/restapi/application/JobControllerApiHandlerHelper.java

```diff
@@ -62,7 +62,7 @@
  * @see JobController
  * @see ApplicationApiHandler
  */
-class JobControllerApiHandlerHelper {
+class JobControllerApiHandlerHelper { // TODO jvenstad: Integrate into ApplicationApiHandler
```

Comment:
Why? ApplicationApiHandler is already way too big..

Reply:
Size in itself isn't a problem, and these paths are served by that handler. 
I guess there's no strong reason to move them, though. But I _will_ move the tests, so the API paths are properly unit tested.